src/mscorlib/src/System/Resources/ResourceManager.cs

```diff
@@ -416,11 +416,9 @@ public virtual void ReleaseAllResources()
 
             lock (localResourceSets)
             {
-                IDictionaryEnumerator setEnum = localResourceSets.GetEnumerator();
-
-                while (setEnum.MoveNext())
+                foreach ((_, ResourceSet resourceSet) in localResourceSets)
                 {
-                    ((ResourceSet)setEnum.Value).Close();
+                    resourceSet.Dispose();
```

Comment:
Is the change from Close() to Dispose() compatible change? Close is virtual, and so it can be overridden...

Reply:
Oops, forgot to remove this change before commit.